chemclipse/tests/org.eclipse.chemclipse.msd.identifier.supplier.nist.fragment.test/src/org/eclipse/chemclipse/msd/identifier/supplier/nist/runtime/LinuxWineSupport_DOSDEVICES_2_ITest.java

```diff
@@ -12,56 +12,54 @@
  *******************************************************************************/
 package org.eclipse.chemclipse.msd.identifier.supplier.nist.runtime;
 
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.fail;
+
 import java.io.File;
 import java.io.FileNotFoundException;
 
 import org.eclipse.chemclipse.msd.identifier.supplier.nist.TestPathHelper;
+import org.junit.Before;
+import org.junit.Test;
 
 public class LinuxWineSupport_DOSDEVICES_2_ITest extends AbstractBackgroundTestCase {
 
 	private IExtendedRuntimeSupport runtimeSupport;
 
 	@Override
-	protected void setUp() throws Exception {
+	@Before
+	public void setUp() throws Exception {
 
 		super.setUp();
 	}
 
-	@Override
-	protected void tearDown() throws Exception {
-
-		super.tearDown();
-	}
-
+	@Test
 	public void testConstruct_1() {
 
 		String nistApp = TestPathHelper.getAbsolutePath(TestPathHelper.TESTFILE_WINE_DOSDEVICES_NIST_APPLICATION);
 		try {
 			runtimeSupport = new LinuxWineSupport(new File(nistApp).getParentFile(), parameterBackground);
 			assertNotNull(runtimeSupport);
 		} catch(FileNotFoundException e) {
-			assertTrue("A file not found exception should not occur here.", false);
+			fail("A file not found exception should not occur here.");
```

Comment:
It's better to not fail but rather to throw the exception so stacktrace is not lost.